bus-mapping/src/bytecode.rs

```diff
@@ -126,6 +126,22 @@ impl Bytecode {
         });
         self
     }
+
+    /// Calldata copy
+    pub fn calldatacopy(
+        &mut self,
+        memory_offset: Word,
+        data_offset: Word,
+        length: Word,
+    ) -> &mut Self {
+        self.append(&crate::bytecode! {
+            PUSH32(memory_offset)
+            PUSH32(data_offset)
+            PUSH32(length)
```

Comment:
I think the sequence should be the reverse?

Reply:
Yes, you're right. But I'll remove this part first and leave the busmapping to next pr